data/model/test/test_organization_ai_settings.py

```
@@ -0,0 +1,425 @@
+
+
+class TestInvalidOrganization:
+    """Tests for handling invalid organization names."""
+
+    def test_get_settings_invalid_org(self, initialized_db):
+        """Test getting settings for non-existent org returns None."""
+        settings = get_org_ai_settings("nonexistent-org")
+
+        assert settings is None
+
+    def test_create_settings_invalid_org_raises(self, initialized_db):
```

Ruff (0.14.8)

Unused method argument: `initialized_db`

(ARG002)